gradle/idea.gradle

```diff
@@ -227,7 +227,7 @@ idea {
               <option name="MAIN_CLASS_NAME" value="org.gradle.debug.GradleRunConfiguration" />
               <option name="VM_PARAMETERS" value="" />
               <option name="PROGRAM_PARAMETERS" value="" />
-              <option name="WORKING_DIRECTORY" value="file://$PROJECT_DIR$" />
+              <option name="WORKING_DIRECTORY" value="file://$MODULE_DIR$" />
```

Comment:
Why change this?

Reply:
That slipped in. I was annoyed that I always needed to do this setting manually when trying to run performance tests from IDEA. I will remove it again.